docker/test/integration/cluster/ImageStore.py

```diff
@@ -164,6 +164,11 @@ def __build_minifi_cpp_image_with_nifi_python_processors(self, python_option):
                 COPY SpecialPropertyTypeChecker.py /opt/minifi/minifi-current/minifi-python/nifi_python_processors/SpecialPropertyTypeChecker.py
                 COPY ProcessContextInterfaceChecker.py /opt/minifi/minifi-current/minifi-python/nifi_python_processors/ProcessContextInterfaceChecker.py
                 COPY CreateFlowFile.py /opt/minifi/minifi-current/minifi-python/nifi_python_processors/CreateFlowFile.py
+                RUN mkdir -p /opt/minifi/minifi-current/minifi-python/nifi_python_processors/compute/
+                COPY subtractutils.py /opt/minifi/minifi-current/minifi-python/nifi_python_processors/compute/subtractutils.py
+                RUN mkdir -p /opt/minifi/minifi-current/minifi-python/nifi_python_processors/processors/
+                COPY RelativeImporterProcessor.py /opt/minifi/minifi-current/minifi-python/nifi_python_processors/compute/processors/RelativeImporterProcessor.py
+                COPY multiplierutils.py /opt/minifi/minifi-current/minifi-python/nifi_python_processors/compute/processors/multiplierutils.py
```

Comment:
This looks weird: why do we create the `nifi_python_processors/processors` directory if we then copy the files into `nifi_python_processors/compute/processors` instead?

Reply:
Good point, it was supposed to be `compute/processors`, but I removed it in a08c7d11aace9eef227e241b05ca63159b6dc95c as COPY command already creates the path if it does not exist.